raiden/network/proxies/token_network_registry.py

```diff
@@ -70,18 +74,19 @@ def __init__(
         self.address_to_tokennetwork = dict()
         self.token_to_tokennetwork = dict()
 
-    def token_to_tokennetwork(self, token_address: typing.TokenAddress):
+    def get_token_network(self, token_address: typing.TokenAddress) -> Union[None, str]:
```

Comment:
It's always nicer to write `Union[None, str]` as `Optional[str]`

And it should be `Optional[typing.Address]` here as the address gets converted to a bytestring.